TuneBot/Services/InteractionHandler.cs

```diff
@@ -0,0 +1,58 @@
+﻿using Discord.Commands;
+using Discord.WebSocket;
+using Microsoft.Extensions.Configuration;
+using System;
+using System.Reflection;
+using System.Threading.Tasks;
+using Discord;
+using Discord.Interactions;
+
+namespace TuneBot.Services
+{
+    internal sealed class InteractionHandler
+    {
+        private readonly DiscordSocketClient _discordSocketClient;
+        private readonly InteractionService _commands;
+        private readonly IServiceProvider _serviceProvider;
+
+        public InteractionHandler(
+            DiscordSocketClient discordSocketClient,
+            InteractionService commands,
+            IServiceProvider serviceProvider)
+        {
+            _discordSocketClient = discordSocketClient;
+            _commands = commands;
+            _serviceProvider = serviceProvider;
+
+            _discordSocketClient.InteractionCreated +=  HandleInteraction;
```

Comment:
This event handler setup is in this code twice. Suspect that line 36 is the one we should keep.